docs/Changelog.md

```diff
@@ -14504,3 +14504,66 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain input and output types to high-precision and 8 bit numeric tensors.</dd>
 </dl>
 
+### <a name="SoftmaxCrossEntropyLoss-12"></a>**SoftmaxCrossEntropyLoss-12**</a>
+
+  Loss function that measures the softmax cross entropy
+  between 'scores' and 'labels'.
+  The loss can be described as:
+      L = (l_1, l_2, ..., l_N), where N is the batch_size
+  
+  shape(scores): (N, C) where C is the number of classes, or (N, C, d1, d2,..., dk),
+  	with K >= 1 in case of K-dimensional loss.
+  shape(labels): (N) where each value is 0 <= labels[i] <= C-1, or (N, d1, d2,..., dk),
+  	with K >= 1 in case of K-dimensional loss.
+  
+  The loss for one sample, l_i, can caculated as follows:
+      l_i = -y[i][c][d1][d2]..[dk], where i is the index of classes.
+  or
+      l_i = -y[i][c][d1][d2]..[dk]*weights[c], if 'weights' is provided.
+  
+  where:
+      p = Softmax(scores)
```

Comment:
I check the document of ONNX Softmax. If the input is N-D tensor with shape (N, C, D1, ..., Dk), it will be flattened into a 2-D tensor with shape (N, C * D1 * ... * Dk). I don't quite sure this matches the Softmax needed in Pytorch SoftmaxCrossEntropy.

Reply:
If there is reduction that means output is a scalar, so it doesn't matter. Only when reduction is none the shape matters, I could try to use Reshape op for that

Reply:
Also I think we should use LogSoftmax instead of Log and Softmax

Reply:
In Pytorch output of Softmax is of the same dimension and shape as input.